talaiot/src/test/kotlin/com/cdsap/talaiot/publisher/PushGatewayPublisherTest.kt

```diff
@@ -63,13 +63,12 @@ class PushGatewayPublisherTest : BehaviorSpec() {
                         }
                     }
                     val content = a.body()?.string()
-
                     assert(
-                        content?.contains(":app:assemble{critical=\"false\",instance=\"\",job=\"task\",metric1=\"value1\",metric2=\"value2\",module=\"app\",rootNode=\"false\",state=\"EXECUTED\",task=\":app:assemble\",value=\"100\",workerId=\"\"} 100")
+                        content?.contains(":app:assemble{cacheEnabled=\"false\",critical=\"false\",instance=\"\",job=\"task\",localCacheHit=\"false\",localCacheMiss=\"false\",metric1=\"value1\",metric2=\"value2\",module=\"app\",remoteCacheHit=\"false\",remoteCacheMiss=\"false\",rootNode=\"false\",state=\"EXECUTED\",task=\":app:assemble\",value=\"100\",workerId=\"\"} 100")
```

Comment:
not related to the PR.
When I run `PushGatewayPublisherTest`, it fails with
```
Could not find a valid Docker environment. Please check configuration. Attempted configurations were:
    UnixSocketClientProviderStrategy: failed with exception InvalidConfigurationException (ping failed). Root cause NoSuchFileException (/var/run/docker.sock)
As no valid configuration was found, execution cannot continue
```
Could you point me what setup I have to do?

Reply:
I guess you have Docker in your machine. When you are running the tests `testContainer` requires:
```
ℹ︎ Checking the system...
        ✔ Docker version should be at least 1.6.0
        ✔ Docker environment should have more than 2GB free disk space
```
Maybe it's worth to write something about how to write tests with TestContainers.